ansible/vars/l10k/l10k_public.yml

```diff
@@ -0,0 +1,200 @@
+SITE_HOST: 'l10k.commcarehq.org'
+COMMTRACK_SITE_HOST: 'l10k.commcarehq.org'
+NO_WWW_SITE_HOST: 'l10k.commcarehq.org'
+deploy_env: l10k
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: True
+  cchq_http: True
+  cchq_http_j2me: False
+  cchq_http_redirect: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech2: False
+  riakcs: False
+
+proxy_type: nginx
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+NEW_RELIC_KEY: "{{ secrets.NEW_RELIC_KEY }}"
+run_newrelic_plugin_agent: False
+
+DATADOG_ENABLED: True
+DATADOG_API_KEY: "{{ secrets.DATADOG_API_KEY }}"
+DATADOG_APP_KEY: "{{ secrets.DATADOG_APP_KEY }}"
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '4096m'
+elasticsearch_cluster_name: 'deves'
+#elasticsearch_node_name: '??'
+
+apache2_key_file: '../config/{{ deploy_env }}/ssl/l10k.commcarehq.org.key'
+apache2_cert_file: '../config/{{ deploy_env }}/ssl/l10k.commcarehq.org.crt'
+
+nginx_combined_cert_file: '../config/{{ deploy_env }}/ssl/l10k.commcarehq.org.combined.crt'
+
+keystore_file: '../config/DimagiKeyStore'
+
+backup_blobdb: False
+backup_postgres: False
+backup_es: False
+backup_couch: False
+postgres_s3: False
+blobdb_s3: False
+couch_s3: False
+
+postgres_users: "{{ secrets.POSTGRES_USERS }}"
+
+postgres_config:
+  max_connections: 300
+  pgb_max_connections: 400
+  pgb_default_pool: 290
+  pgb_reserve_pool: 5
+  pgb_pool_timeout: 1
+  pgb_pool_mode: transaction
+
+formplayer_db_name: formplayer
+
+postgresql_dbs:
+  - name: "{{ localsettings.UCR_DATABASE_NAME }}"
+  - name: "{{ localsettings.PG_DATABASE_NAME }}"
+    django_alias: default
+  - name: "{{ formplayer_db_name }}"
+
+couch_dbs:
+  default:
+    host: 10.162.36.254
+    port: 5984
+    name: commcarehq
+    username: "{{ localsettings_private.COUCH_USERNAME }}"
+    password: "{{ localsettings_private.COUCH_PASSWORD }}"
+    is_https: False
+
+redis_max_memory: 4096mb
+redis_logfile: '/var/log/redis/redis-server.log'
+redis_syslog_enabled: 'no'
+redis_database_save_times: []
+redis_max_clients: 10000
+redis_maxmemory_policy: allkeys-lru
+
+ECRYPTFS_PASSWORD: "{{ secrets.ECRYPTFS_PASSWORD }}"
+
+KSPLICE_ACTIVE: yes
+KSPLICE_ACTIVATION_KEY: "{{ secrets.KSPLICE_ACTIVATION_KEY }}"
+
+AMQP_USER: "{{ secrets.AMQP_USER }}"
+AMQP_PASSWORD: "{{ secrets.AMQP_PASSWORD }}"
+AMQP_HOST: "10.162.36.254"
+AMQP_NAME: commcarehq
+
+shared_drive_enabled: false
+
+localsettings:
+  ALLOWED_HOSTS:
+    - l10k.commcarehq.org
+    - 10.162.36.254
+    - localhost
+    - 127.0.0.1
+    - testserver
```

Comment:
"testserver" is in every _public.yml vars, i know that doesn't answer the question, i am just saying, it's popular